includes/wikia/DefaultSettings.php

```diff
@@ -1797,3 +1798,8 @@
  * flag for using (possibly external) service for managing a user's preferences
  */
 $wgPreferencesUseService = false;
+
+/**
+ * Parser Tag
+ */
+require_once "$IP/extensions/wikia/SoundCloudTag/SoundCloudTag.setup.php" ;
```

Comment:
It seems there are a very small number of other extension inclusions in wikia/DefaultSettings... and one of them even has a TODO that says "move this to commonextensions".  I agree that since most extensions are enabled globally with a variable in Default settings, this is a better practice, even though it's an otherwise useless/unused var.